cfgov/v1/wagtail_hooks.py

```diff
@@ -19,6 +19,11 @@
 logger = logging.getLogger(__name__)
 
 
+@hooks.register('before_delete_page')
+def raise_delete_error(request, page):
+    raise Exception('Deletion via POST is disabled')
```

Comment:
Would it be better to raise [`django.core.exceptions.PermissionDenied`](https://docs.djangoproject.com/en/1.8/ref/exceptions/#permissiondenied) here? This triggers a 403 error instead of a 500. It's also the same exception class [that Wagtail raises](https://github.com/wagtail/wagtail/blob/v1.7/wagtail/wagtailadmin/views/pages.py#L491) if someone tries to delete but doesn't have the correct permissions.